### ISSUE TITLE
Polls

### ISSUE THREAD

**Issue by jakewb-b** [vector-im/roadmap#29]
***Summary***
A simple poll feature native to Element clients, allowing users to ask questions with multiple-choice answers.

***Intended Outcome***
Native poll functionality in Element web and mobile, with the ability to ask a question with up to 20 pre-defined responses. 
Basic functionality displays results as soon as a user has voted, and only allows one vote per user.
Advanced functionality for future state includes options such as hidden results, where results are only revealed after the poll ends, and multi-selection of responses.

**Comment by MartinDelille** [comment]
Allowing multi-selection polls has been announced here: https://element.io/blog/element-welcomes-polls/

Is there any other issue somewhere that I could follow ?

**Comment by JPustkuchen** [comment]
I agree multi-selection polls is the most important feature now in polls area!

Update: Here we go: https://github.com/element-hq/element-meta/issues/2694
